package.json

```diff
@@ -15,6 +15,7 @@
     "@react-navigation/stack": "^5.14.4",
     "react": "17.0.1",
     "react-native": "0.64.0",
+    "react-native-card-stack-swiper": "^1.2.5",
```

Comment:
Glad we went with this package instead of doing it manually. The code that lives in these components are way more intense thatn I would have thought!